testing/maas_billing_tests_independent/tests/test_streaming_chat.py

```
@@ -0,0 +1,103 @@
+        try:
+            j = json.loads(data)
+            saw_json = True
+            ch = (j.get("choices") or [{}])[0]
+            # OpenAI-style SSE frames: choices[].delta.{content|role}
+            delta = ch.get("delta") or (ch.get("message") if "message" in ch else {}) or {}
+            piece = delta.get("content") or ""
+            if isinstance(piece, str) and piece:
+                chunks.append(piece)
+                if DEBUG:
+                    print(f"[chunk {len(chunks):02d}] {piece!r}")
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)